Library/Homebrew/requirements/php_requirement.rb

```diff
@@ -0,0 +1,11 @@
+class PhpRequirement < Requirement
+  fatal true
+  satisfy { which "php" }
+
+  env do
+    if which("php") != "/usr/bin/php"
```

Comment:
That's not the case for `Requirements`; they will example your custom PATH in this `env` block.